source/cloud/windows.md

```diff
@@ -33,33 +38,68 @@ Click or select the `Overview` tab, you will see the Windows admin password
 
 ![image](img/windows_admin_pass.png)
 
-You should use this password to login the first time in your Windows machine.
-(You must reset the admin password to connect via Remote Desktop).
-
-## Reset Windows admin password
-
-Click on `Console` tab to get access to your Windows machine via VNC. If you see a black
-screen just click and drag with your mouse to unlock the screen.
-You will see a Windows login screen similar to this:
-
-![image](img/windows_login.png)
-
-You should type the same password displayed from the Metadata in the previous section.
-Windows will ask you to reset the admin password, type a new password twice to verify
-that is correct.
-
-:::{tip}
-Usually you have to reset the password after the first boot or if you change
-the VM's flavor. In that case just use the original password provided
-by the metadata and reset the password again.
-:::
-
 At this point you can use Windows Remote Desktop (RDP) with your new admin user and password
 to connect to your VM.
-Make sure you use the correct port to connect to your Windows Remote Desktop,
+:::{note}
+Remember: use `terraform output` to get the connection details for your VM
 (please see [Terraform section](terraform.md) for more details).
+:::
 
 :::{note}
 Take into account you should upgrade your Windows OS after the first boot.
 You should activate your Windows machine with a valid key as well.
 :::
+# Creating your own Windows image
+It is possible to customize and upload your own windows image. 
+The guide below is a semi-automated method of doing so. 
+## Preperation
+1) Download the latest stable virtio driver:
+https://fedorapeople.org/groups/virt/virtio-win/direct-downloads/stable-virtio/
+3) Save as `virtio-win.iso`
+4) Download the latest windows iso, save as `Windows.iso`
+* [Win10](https://www.microsoft.com/en-us/software-download/windows10ISO)
+* [Win11](https://www.microsoft.com/software-download/windows11)
```

Comment:
I would not include windows 11 Image as example until we do not have RHOSP 17.1 deployed that supports it.